coin/coins.yml

```diff
@@ -92,7 +92,7 @@
   name: IoTeX Network Mainnet
   decimals: 18
   blockTime: 10000
-  blockchain: IoTeX Network Mainnet
+  blockchain: Ethereum
```

Comment:
I'm not sure,  shouldn't add a new Iotex chain rather than changing the current chain?

Reply:
Should be a new chain

Reply:
Should be iotexevm with a coin id 10004689
 https://github.com/trustwallet/wallet-core/blob/master/registry.json#L1735

Reply:
@iCell this change is correct. There is another iotex coin already exists in this file.

Reply:
@iCell yes we have iotex and iotex evm, previously I added iotex evm incorrectly, now fixed